packages/query/src/queries/get-twitter-metadata.graphql

```diff
@@ -0,0 +1,31 @@
+query GetTwitterMetadata($uri: String!) {
+	nodeByUri(uri: $uri) {
```

Comment:
1. We use `templateByUri`
2. Use fragments and interfaces `... on NodeWithFeaturedImage`
3. Every node should have the global `id` queried, so things are correctly cached by the fewhen possible.